src/app/page.tsx

```diff
@@ -1,15 +1,10 @@
-import { Locale } from '@/../i18n.config'
 import { getDictionary } from '@/lib/dictionary'
 import LandingPage from '@/components/LandingPage'
 
-export default async function Home({
-  params: { lang },
-}: {
-  params: { lang: Locale }
-}) {
+export default async function Home() {
   const {
     home: { heroSection: dict },
-  } = await getDictionary(lang)
+  } = await getDictionary('en')
```

Comment:
we shouldn't need a `dictionary` anymore right?
- all content should be the default english variation